lib/logflare/sources/rules.ex

```diff
@@ -38,7 +37,9 @@ defmodule Logflare.Rules do
     |> Repo.insert()
     |> case do
       {:ok, %Rule{backend_id: backend_id} = rule} = result when backend_id != nil ->
-        if Backends.source_sup_started?(rule.source_id), do: SourceSup.start_rule_child(rule)
+        # expected to fail on nodes where SourceSup is not started
+        :rpc.multicall(SourceSup, :start_rule_child, [rule])
```

Comment:
just a reminder that the default timeout is set to infinity

Reply:
refactored to add a default timeout, thanks for the catch 🙏